crates/sc-subspace-chain-specs/src/lib.rs

```diff
@@ -18,3 +18,4 @@
 
 /// Devnet chain spec
 pub const DEVNET_CHAIN_SPEC: &str = include_str!("../res/chain-spec-raw-devnet.json");
+pub const TAURUS_CHAIN_SPEC: &str = include_str!("../res/chain-spec-raw-taurus.json");
```

Comment:
New line at the end of the file, please, like it was before. Ideally configure IDE to add it automatically.